src/main/java/pl/fracz/mcr/source/Line.java

```diff
@@ -0,0 +1,117 @@
+package pl.fracz.mcr.source;
+
+/*
+ 2013-10-23, fracz, first implementation
+ 2013-10-30, fracz, added syntax highlighting
+ 2014-02-26, fracz, added ability to add voice comment
+ */
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.graphics.Color;
+import android.graphics.Typeface;
+import android.text.Html;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import pl.fracz.mcr.comment.AbstractComment;
+import pl.fracz.mcr.comment.Comment;
+import pl.fracz.mcr.comment.CommentNotAddedException;
+
+import java.io.File;
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * View that represents one line of code.
+ */
+@SuppressLint("ViewConstructor")
+public class Line extends LinearLayout implements Serializable {
+    private static final long serialVersionUID = 3076583280108678995L;
+    private static final int TWO = 2;
+
+    private final int _lineNumber;
+
+    private final String _lineOfCode;
+
+    // holds the line number
```

Comment:
Prawdopodobnie komentarz który kłamie, ponieważ sugeruje że to liczba, a to jest TextView